package.json

```diff
@@ -77,7 +77,7 @@
         "build": "npm run build:pack && npm run build:vue && npm run build:illustration-translations",
         "dependencies:postinstall": "php -r \"file_put_contents('.package.hash', sha1_file('package-lock.json'));\"",
         "dependencies": "npm run dependencies:postinstall",
-        "postinstall": "patch-package --patch-dir tools/patches/npm",
+        "postinstall": "patch-package --patch-dir tools/patches/npm && ./tests/e2e/scripts/sync-playwright-version.sh",
```

Comment:
Someone will have to commit this manually everytime the library is updated. Could you try to propose a workflow that will do it automatically, like we do in the `bump_version_after_release` workflow?

Reply:
I tried to look at its code and I did not found the answer. If I had to build such a tool, I would use the `noscripts` option everywhere, to prevent custom scripts to fail, for instance if they are not made to be executed in an unexpected env.